frontends/election-manager/src/components/printable_area.tsx

```diff
@@ -0,0 +1,26 @@
+import React, { useEffect, useRef } from 'react';
+import ReactDom from 'react-dom';
+
+interface Props {
+  children?: React.ReactNode;
+}
+
+/**
+ * <PrintableArea> renders the provided children in a standalone div with the .print-only class
+ * name (standalone meaning via a React portal to ensure that a parent's CSS styles don't prevent
+ * printability)
+ */
+export function PrintableArea({ children }: Props): JSX.Element {
```

Comment:
Previously, printable content had to be rendered outside the relevant `Screen` component to avoid the `Screen`'s `@media print { display: none; }` [style](https://github.com/votingworks/vxsuite/blob/a354ea74fb27937b33897a1f7fcf775fdb5fb5fb/frontends/election-manager/src/components/screen.tsx#L20-L22). This meant that print buttons and the corresponding printable content often couldn't be colocated in code, e.g. [print button](https://github.com/votingworks/vxsuite/blob/a354ea74fb27937b33897a1f7fcf775fdb5fb5fb/frontends/election-manager/src/screens/tally_report_screen.tsx#L226-L233) and [printable content](https://github.com/votingworks/vxsuite/blob/a354ea74fb27937b33897a1f7fcf775fdb5fb5fb/frontends/election-manager/src/screens/tally_report_screen.tsx#L268-L281). Now, these can be colocated and even placed together in sub-components, as you'll see below with the `ZeroReportPrintButton` component!

Reply:
This is great! 👍🏼